runtime/src/fips.rs

```diff
@@ -80,14 +59,47 @@ impl FipsVersionCmd {
     }
 }
 
-pub struct FipsSelfTestCmd;
-impl FipsSelfTestCmd {
+pub mod fips_self_test_cmd {
```

Comment:
nit: this name is now inconsistent with the rest. Either change all or change none.

Reply:
this is rust convention for modules.

Reply:
agree, but then change them all instead of just changing this one

Reply:
sorry, ignore. The other one I was looking at is a struct, not mod.